### PR TITLE
sesify-viz - bump dep for visualization enhancement

### PR DESCRIPTION
**Pull request by kumavis**

### CODE REVIEW

**Comment by kumavis** [comment]
repo now includes the react app https://github.com/kumavis/sesify-viz

**Comment by kumavis** [comment]
hmm not ready yet

**Comment by kumavis** [comment]
fyi got a random failure in `test-e2e-firefox`

File: package.json [hunk 1]
```diff
@@ -270,7 +270,7 @@
     "sass-loader": "^7.0.1",
     "selenium-webdriver": "^3.5.0",
     "sesify": "^4.2.1",
-    "sesify-viz": "^2.0.1",
+    "sesify-viz": "3.0.5",
```
Comment (Gudahtt) [inline package.json]:
```suggestion
    "sesify-viz": "^3.0.5",
```
Could this be left as a range, or does it need to be pinned here?

Maybe less important given that this is a tool that's used independently, but as a general rule ranges allow for better de-duplication and make life easier for downstream projects.

**Comment by Gudahtt** [comment]
> fyi got a random failure in `test-e2e-firefox`

Both of the e2e tests suffer from ~3 different intermittent failures, sadly. Not sure on the root cause yet.

**Review by Gudahtt — DISMISSED** [review]

**Comment by Gudahtt** [comment]
Nice - I appreciate the package metadata being in the top-left now. It also seems to persist if you mouse away quickly, which is useful.

I did get an error when clicking on one of the other tabs though:
```
ReferenceError: triggerGraphUpdate is not defined
```

File: package.json (hunk 1 above)
Comment (kumavis) [reply package.json]: yeah yarn gave me this when I specified a version (to make sure it updated). I don't think npm behaved this way.

**Comment by kumavis** [comment]
fixed the bug in the viz app
yeah its still not very good, but just beginning
give me feature requests

**Review by Gudahtt — DISMISSED** [review]
Looks good! No more error.

**Comment by kumavis** [comment]
fixed a `yarn.lock` conflict

**Review by Gudahtt — APPROVED** [review]

**Pull request merged**